remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -384,12 +389,30 @@
             }
         },
         {
+            "comments": "Security rules used for the VM network interface",
             "type": "Microsoft.Network/networkSecurityGroups",
             "name": "[variables('networkSecurityGroupName')]",
             "apiVersion": "[variables('networkApiVersion')]",
             "location": "[variables('location')]",
             "tags": {
                 "IotSuiteType": "[parameters('solutionType')]"
+            },
```

Comment:
do we need 80 open?  I don't think so - this would be for internal service communication only from the site and from postman (if auth is off), yes?

Reply:
Yes if we want to hit http through postman with auth off then we will need to enable port 80. I can add how to do that in documentation as well, so that we are more secure be default.